src/ko/index.html

```diff
@@ -302,12 +302,12 @@ <h1>피카츄 배구 - 슈퍼 AI 버전</h1>
 					</p>
 					<hr style="width: 100%;">
 					<p class="reverse-engineered-by">
-						<a href="https://gorisanson.github.io/pikachu-volleyball/ko/" target="_blank">웹 버전</a>을 바탕으로
-						<a href="https://gorisanson.github.io/" target="_blank">이규태</a>가 개발한 것입니다.
+						<a href="https://gorisanson.github.io/pikachu-volleyball/ko/" target="_blank">피카츄 배구
+						웹 버전</a>(만든 사람: <a href="https://gorisanson.github.io/" target="_blank">이규태</a>)에 기반합니다.
```

Comment:
The original Korean expression was read like "(This Super AI version is) developed by Kyutae Lee based on the web version", which is apparently misleading. It was developed by you, DukLL. So I changed the Korean expression to be read as "(This Super AI version is) based on the Pikachu Volleyball web version (which is made by Kyutae Lee)."